backend/lightning/lightning.go

```diff
@@ -72,14 +75,24 @@ func (lightning *Lightning) Activate() error {
 		return err
 	}
 
+	fingerprint, err := keystore.RootFingerprint()
+	if err != nil {
+		return err
+	}
+
 	entropyMnemonic, err := bip39.NewMnemonic(entropy)
 	if err != nil {
 		lightning.log.WithError(err).Warn("Error generating mnemonic")
 		return errp.New("Error generating mnemonic")
 	}
 
+	lightningAccount := config.LightningAccountConfig{
+		Mnemonic:        entropyMnemonic,
+		RootFingerprint: fingerprint,
+		Code:            types.Code(strings.Join([]string{"v0-", hex.EncodeToString(fingerprint), "-ln-0"}, "")),
+	}
 	lightningConfig.Inactive = false
-	lightningConfig.Mnemonic = entropyMnemonic
+	lightningConfig.Accounts = append(lightningConfig.Accounts, &lightningAccount)
```

Comment:
shouldn't we check if the account already exists?

Reply:
Activate should be called only once, when we enable lightning in the settings, and the config is cleaned up on `Deactivate()`, so it should not be possible to have any account already saved here.

Reply:
Sure, but I think it can't hurt to do a check anyway, because it's not easy to see that this property holds and will hold - plus sometimes there are bugs or race conditions etc.

Similar check exists here for normal accounts:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/b02e97fd6492bf8163289e307c607ec6a03227f9/backend/accounts.go#L654-L657

Reply:
Added a check to verify that the accounts list is empty on `Activate()`